lib/daru/dataframe.rb

```diff
@@ -2004,6 +2004,23 @@ def split_by_category cat_name
       end
     end
 
+    # returns array of row tuples at given index(s)
+    def access_row_tuples_by_indexs *indexes
```

Comment:
let me know If I can use `access_row` method to get row tuples or I can reorder the `access_row` output vectors.